Running custom power spectrum generator...
venv/bin/python -m power_spectrum_generation.custom_power_spectrum pycosmo_input_512.param
Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/mnt/c/Users/kitty/PyCosmo_SimGadget/PyCosmo_Power_Spectrum/power_spectrum_generation/custom_power_spectrum.py", line 96, in <module>
    main(args.param_file)
  File "/mnt/c/Users/kitty/PyCosmo_SimGadget/PyCosmo_Power_Spectrum/power_spectrum_generation/custom_power_spectrum.py", line 81, in main
    run_spectrum_generation(parameters)
  File "/mnt/c/Users/kitty/PyCosmo_SimGadget/PyCosmo_Power_Spectrum/power_spectrum_generation/custom_power_spectrum.py", line 51, in run_spectrum_generation
    power_spectrum = PowerSpectrumClass(parameters)
  File "/mnt/c/Users/kitty/PyCosmo_SimGadget/PyCosmo_Power_Spectrum/power_spectrum_generation/power_spectrum_class.py", line 74, in __init__
    print(self.cosmo.pk_norm_type)
AttributeError: 'Cosmo' object has no attribute 'pk_norm_type'
Using parameter file: pycosmo_input_512.param
Nsample: 512
Box: 50000
Omega: 0.3
OmegaLambda: 0.7
OmegaBaryon: 0.0
HubbleParam: 0.7
Redshift: 0
Sigma8: 0.9
LinearFittingFunction: 0
NonLinearFittingFunction: 0
InputSpectrum_UnitLength_in_cm: 3.085678e24
ReNormalizeInputSpectrum: 1
UnitLength_in_cm: 3.085678e21
UnitMass_in_g: 1.989e43
UnitVelocity_in_cm_per_s: 1e5
[main]
model = LCDM
core_equations_files = ['/mnt/c/Users/kitty/PyCosmo_SimGadget/PyCosmo_Power_Spectrum/venv/lib/python3.9/site-packages/PyCosmo/CosmologyCore.py']
compilation_flags = -O3
default_ini_file = config/default_lcdm.ini

[equation_parameters]
l_max = 20
splits = None
reorder = True
cache hit at /home/kitty/_cache/PyCosmo/linux-x86_64/2_2_0/2_2_4__87c12b.pkl
load wrapper from /home/kitty/_cache/sympy2c/linux-x86_64/2_2_4__np_1_26_4/1db8b055_fc3ec
Parameters updated
Parameters updated
Parameters updated
make: *** [Makefile:127: run] Error 1
